decidim-core/app/models/decidim/participatory_process.rb

```diff
@@ -8,7 +8,7 @@ module Decidim
   class ParticipatoryProcess < ApplicationRecord
     belongs_to :organization, foreign_key: "decidim_organization_id", class_name: Decidim::Organization
     has_many :steps, foreign_key: "decidim_participatory_process_id", class_name: Decidim::ParticipatoryProcessStep
-
+    has_one :active_step, -> { where(active: true) }, foreign_key: "decidim_participatory_process_id", class_name: Decidim::ParticipatoryProcessStep
```

Comment:
Not sure I like associations defined like this as can give the bad impression they can be modified, and this implementation currently only works one way.

I'd like something like:

```ruby
def active_step
  steps.where(active: true)
end
```

Alternatively, you could add `active_step` to this model's list of `attr_readonly` attributes:

```ruby
attr_readonly :active_step
```